react/Spinner/Readme.md

```diff
@@ -0,0 +1,49 @@
+Show a spinner when loading something.
+
+### Default 
+
+```
+<Spinner />
+```
+
+### Color
+
+```
+<div>
+  blue: <Spinner color='blue' />
+  grey: <Spinner color='grey' />
+  white: <Spinner color='white' />
+  red: <Spinner color='red' />
+</div>
+```
+
+### Placement
+
+```
+<div>
+<Spinner noMargin={ true }/>
+</div>
+```
```

Comment:
I see no difference with `noMargin={true}`, see below:

![](https://i.imgur.com/VxonbV4.png)

Reply:
Bug from the spinner

Reply:
https://github.com/cozy/cozy-ui/issues/197